js/src/product-feed/product-statistics/status-box/sync-product-statistics.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import Status from '.~/product-feed/product-statistics/status-box/status';
+import ErrorIcon from '.~/components/error-icon';
+
+/**
+ * @typedef {import('.~/data/actions').ProductStatistics } ProductStatistics
+ */
+
+/**
+ * Renders status information for the Product Sync
+ *
+ * @param {Object} props The component props
+ * @param {Function} props.refreshStats
+ * @param {string} props.error
+ * @return {JSX.Element} The status for the Product Sync
+ */
+function SyncProductStatistics( { refreshStats, error } ) {
+	return (
+		<Status
+			title={ __( 'Overview Stats:', 'google-listings-and-ads' ) }
+			icon={ <ErrorIcon size={ 24 } /> }
+			label={
+				<Button
```

Comment:
What about using [AppButton](https://github.com/woocommerce/google-listings-and-ads/blob/develop/js/src/components/app-button/index.js) so we can also add the `eventName` to it for tracking.